gravitee-apim-console-webui/src/management/api/api-traffic-v4/analytics/components/api-analytics-widget-table/api-analytics-widget-table.component.html

```diff
@@ -40,7 +40,9 @@
                 {{ element | percent }}
               }
               @default {
-                {{ element }}
+                <span [ngClass]="{ 'empty-cell': element === '' }">
+                  {{ element === '' ? '(empty)' : element }}
+                </span>
```

Comment:
I agree this needs to be simplified. I would have gone about it like this to avoid checking if it's empty twice:
```suggestion
 @if (element?.length) {
    <span>{{ element }}</span>
 } @else {
   <span class="empty-cell">(empty)</span>
 }
```

Reply:
simplification hell ;-)